projects/fsastorefrontlib/src/cms-components/sync-pilot/sync-pilot-connection/sync-pilot-connection.component.ts

```diff
@@ -15,9 +15,13 @@ import {
   EGender,
   GuestInfo,
 } from '@syncpilot/bpool-guest-lib/models/guestinfo';
-import { SyncPilotGender } from '../../occ/occ-models/occ.models';
-import { SyncPilotDialogComponent } from '../sync-pilot-dialog/sync-pilot-dialog.component';
-import { CMSConnectionComponent } from '../../occ/occ-models/cms-component.models';
+import { SyncPilotGender } from '../../../occ/occ-models/occ.models';
+import { SyncPilotDialogComponent } from '../../sync-pilot-dialog/sync-pilot-dialog.component';
+import { CMSConnectionComponent } from '../../../occ/occ-models/cms-component.models';
+import {
+  GROUP_ID,
+  OWNER_ID,
+} from '../../../core/sync-pilot-config/sync-pilot-config';
 
 @Component({
   selector: 'cx-fs-sync-pilot-connection-component',
```

Comment:
Since this is a CMS component, maybe it would be better to rename it to `cx-fs-sync-pilot-cms-component` and the other one to `cx-fs-sync-pilot-generic-component` or something similar to make really clear their usage?

Reply:
I'll rename them.